### DISCUSSION TITLE
Welcome to xray-maven-plugin Discussions!

### DISCUSSION THREAD

**Discussion by bitcoder** [Xray-App/xray-maven-plugin#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MaherAccor** [comment]
Hello Sergio ,
I hope you're well!
I had a problem importing a test on xray :
I can import the tests on xray but I can't associate the testExec to a testPlan .
I want to associate the testExec to TestPlan with the format Cucumber but doesn't works .
This my conf :
 <groupId>app.getxray</groupId>
                <artifactId>xray-maven-plugin</artifactId>
                <version>0.7.6</version>
                <configuration>
                    <clientId>5887C6E90BA14888ADD5677B792F1A21D</clientId>
                    <clientSecret>6c37c9cb2zsd813b2b2c1babd496bc1cd7e1423640cb37399908db065ad09a874</clientSecret>
                    <cloud>true</cloud>
                    <reportFormat>cucumber</reportFormat>
                    <projectKey>DAPP</projectKey>
                    <testPlanKey>DAPP-38120</testPlanKey>
                    <reportFile>target/report-cucumber.json</reportFile>

**Reply by bitcoder** [reply]
Please create a separate discussion for this. Thanks :)

**Comment by fredericMagnenat** [comment]
Hey,

I am using the xray-maven-plugin version 0.8.0 to import JUnit test results into Jira Xray. However, I have noticed that the plugin is calling an outdated API endpoint.

Currently, the plugin uses rest/raven/2.0/import/execution/junit?projectKey=XXX, but Xray's documentation recommends using the newer endpoint: rest/raven/2.0/api/import

The plugin doesn’t seem to provide a way to configure the endpoint to use this new API path. I would like to know how to resolve this issue or if there’s a way to customize the API URL in the plugin’s configuration.

Any help would be appreciated.

Any help would be appreciated.

**Reply by bitcoder** [reply]
Please create a separate discussion for this. I'll answer there. Thanks :)
